lib/tasks/demo.rake

```diff
@@ -1,6 +1,41 @@
 # frozen_string_literal: true
 
 namespace :demo do
+desc 'Backup the database'
+  task backup_database: :environment do
+    raise 'This task is only for use in a development environment' unless Rails.env == 'development' || ENV['DISABLE_DATABASE_ENVIRONMENT_CHECK']
+    username = ActiveRecord::Base.configurations.configurations[1].config['username']
+    database = ActiveRecord::Base.configurations.configurations[1].config['database']
+    system "mysqldump --opt --user=#{username} #{database} > sara_database_#{Time.now.to_i}.sql"
+  end
+
+  desc 'Restore the database'
+  task restore_database: :environment do
+    # Example usage: rake demo:restore_database FILE=sara_database_1606835867.sql
+    raise 'This task is only for use in a development environment' unless Rails.env == 'development' || ENV['DISABLE_DATABASE_ENVIRONMENT_CHECK']
+    username = ActiveRecord::Base.configurations.configurations[1].config['username']
+    database = ActiveRecord::Base.configurations.configurations[1].config['database']
+    system "mysql --user=#{username} #{database} < #{ENV['FILE']}"
```

Comment:
can you add a validation for `ENV['FILE']` before the mysql call? Maybe something like `return && puts 'FILE environment variable must be set to run this task' if ENV['FILE'].nil?`.

Right now if it's excluded someone might not know what to set without having to look at the actual code.

Reply:
Good suggestion -- done